dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/subcommands/impl/All.java

```diff
@@ -112,17 +112,17 @@ protected void forkRepositoriesFound(Multimap<String, String> pathToDockerfilesI
                 log.warn("Skipping {} because it's a fork already. Sending a PR to a fork is unsupported at the moment.",
                         parentRepoName);
             } else {
-                pathToDockerfilesInParentRepo.put(parentRepoName, c.getPath());
-                imagesFoundInParentRepo.put(parentRepoName, image);
-
-                // fork the parent if not already forked or we couldn't fork
-                if (!parentReposAlreadyChecked.contains(parentRepoName)) {
+                // fork the parent if not already forked
+                if (!parentReposForked.contains(parentRepoName)) {
```

Comment:
Could you make this a positive assertion? 
```suggestion
                if (parentReposForked.contains(parentRepoName) == false) {
```

Reply:
@justinharringa Updated PR.